### ISSUE TITLE
Create a get_utilization() method on the RIR model

### ISSUE THREAD

**Issue by jeremystretch** [netbox-community/netbox#4961]
### Proposed Changes
Create a `get_utilization()` method on the RIR model to return the total utilization of all assigned aggregates, similar to what exists on the Prefix model.

### Justification
RIR utilization is currently calculated by manipulating the queryset used in the list view. This is both inefficient and inflexible, because no convenient method exists for determining the utilization of a particular RIR. Moving this logic into a method is cleaner and allows us to remove the `alter_queryset()` method on ObjectListView.

**Comment by jeremystretch** [comment]
Blocked by #1846

**Comment by jeremystretch** [comment]
Assigning myself to further investigate the feasibility of this. If found to be more trouble than it's likely worth, I'll open a separate issue to deprecate the RIR statistics feature entirely.
